File: bench/repos/mini_ts_support_service/tests/public_checks.ts
import assert from "node:assert/strict";

import { supportEmailDomain } from "../support.ts";

assert.equal(
  supportEmailDomain({ contacts: { supportEmail: "ops@example.com" } }),
  "example.com",
);
assert.equal(
  supportEmailDomain({ contacts: { supportEmail: "HELP@Travel.test" } }),
  "travel.test",
);

File: bench/repos/mini_py_support_service/tests/public_checks.py
from pathlib import Path
import sys

sys.path.insert(0, str(Path(__file__).resolve().parents[1]))

from support import support_email_domain

assert support_email_domain({"contacts": {"support_email": "ops@example.com"}}) == "example.com"
assert support_email_domain({"contacts": {"support_email": "HELP@Travel.test"}}) == "travel.test"
